features/FxA-46-sms-install-link/README.md

```diff
@@ -0,0 +1,61 @@
+# SMS Install Link
+
+https://mozilla.aha.io/features/FXA-46
+
+## Stories
+
+As a user signing up for a Firefox Account, I may want to easily install
+Firefox on my Android or iOS device, removing the friction of searching
+by sending me the link in a text message.
+
+## Success Criteria
+
+This feature will be successful if we measurably increase the rate of
+Firefox activations for Android and iOS.
+
+A metrics funnel we can use to measure success is:
+
+1. Number of users who submit their number to the form in order to
+   receive a SMS
+2. Number of users who click the link
+3. Number of users who install Firefox from this link (utm_campaign
+   thingies?)
+4. Number of users who connect a second device after having requested a
+   link
+
+We can optionally generate a unique URL for every user, such
+`https://get.firefox.com/hash(uid)`, to improve the accuracy of our
+metrics, but it is not required.
+
+## Details
+
+On Firefox Desktop, after a user successfully signs up to a new account,
+we will show a small form prompting the user to enter their telephone
+number to be texted a link to install Firefox Mobile.
+
+The SMS capability currently exists in the [Basket API][], allowing us
+to essentially SMS a user via `POST /news/subscribe_sms`. It is used for
+the Firefox for Android landing page, and can be viewed here (it does
+detect if you're in the US before showing the form):
+https://mozilla.org/firefox/android
+
+
+This form would need to be only shown to those in the US, as our current
+provider behind Basket only allows US phone numbers.
+
+- UX design of form to show to user
+- Implementation of form in content-server
+- UX of text message wording
+- Short URL pointing to page that redirects based on OS
+  - Play Store for Android
+  - App Store for iOS
+- Work in Basket repo to customize text message with our link
+
+## Unresolved Questions
+
+- Should this be an experiment, only being shown to a percentage of
+  users?
+- Should we generate a unique short URL for every user, or just use a
+  single URL?
```

Comment:
If we can get a good metrics story without having to customize the URL, I think we should do that as the first iteration.  It will substantially reduce uncertainty/risk by removing the need to ship new code in basket.  If the feature is successful, customizing things more for each user will be a good follow-up.